deployments/anvildev/sources.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Having to maintain this file is pretty much a non-starter. If it exists to enable generating a count for obsoleted and popped snapshots, then that would be too much of a burden. It's OK to only generate counts for the currently configured snapshots.

In the long run, we should encode the size (# of subgraphs) of a source in the spec and compute the partition prefix length dynamically. We'd have to figure out what to do for sandbox deployments that use a common prefix for some of the snapshots.

Whatever solution we come up with in this PR should also be extended to HCA but it would be OK to do that in a separate issue/PR.

Reply:
Resolved at PL.